src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -531,4 +534,32 @@ class HttpGoogleServicesDAO(
       }
     }
   }
+
+  private def executeGoogleRequest[T](request: AbstractGoogleClientRequest[T]): T = {
+    if (logger.underlying.isDebugEnabled) {
+      val payload = Option(request.getHttpContent) match {
+        case Some(content: JsonHttpContent) => content.getData.toString
+        case Some(content) if content.getLength == 0 => "[none]"
+        case None => "[none]"
+        case _ => "[unknown]"
+      }
+
+      val start = System.currentTimeMillis()
+      Try {
+        request.executeUnparsed()
+      } match {
+        case Success(response) =>
+          logger.debug(s"GOOGLE REQUEST: ${request.getRequestMethod} ${request.buildHttpRequestUrl()}, payload: ${payload}, SUCCESS http status code: ${response.getStatusCode}, time ${System.currentTimeMillis() - start} ms")
+          response.parseAs(request.getResponseClass)
+        case Failure(httpRegrets: HttpResponseException) =>
+          logger.debug(s"GOOGLE REQUEST: ${request.getRequestMethod} ${request.buildHttpRequestUrl()}, payload: ${payload}, FAILED http status code: ${httpRegrets.getStatusCode}, time ${System.currentTimeMillis() - start} ms")
+          throw httpRegrets
+        case Failure(regrets) =>
+          logger.debug(s"GOOGLE REQUEST: ${request.getRequestMethod} ${request.buildHttpRequestUrl()}, payload: ${payload}, EXCEPTION message: ${regrets.getMessage}, time ${System.currentTimeMillis() - start} ms")
```

Comment:
Idea: can (should?) we make these JSON?

(might be a dumb idea, requesting feedback)